modules/core/04-channel/keeper/grpc_query.go

```diff
@@ -398,7 +398,7 @@ func (q Keeper) UnreceivedPackets(c context.Context, req *types.QueryUnreceivedP
 
 	ctx := sdk.UnwrapSDKContext(c)
 
-	channel, found := q.GetChannel(sdk.UnwrapSDKContext(c), req.PortId, req.ChannelId)
+	channel, found := k.GetChannel(sdk.UnwrapSDKContext(c), req.PortId, req.ChannelId)
```

Comment:
Seems like this existed from before but I guess it wouldn't be a bad idea to fix it now:

```suggestion
	channel, found := k.GetChannel(ctx, req.PortId, req.ChannelId)
```

Reply:
bump don't think this was addressed, no? @crodriguezvega